website/docs/advanced/search_path.md

```diff
@@ -104,23 +105,39 @@ dataset:
 </div>
 </div>
 
+Overriding `dataset=imagenet` from the commandline:
+
+<div className="row">
+<div className="col col--6">
+
+```bash title="command line override"
+python my_app.py dataset=imagenet
+
+
+```
+
+</div>
+
+<div className="col  col--6">
 
+```python title="my_app.py output"
+dataset:
+  name: imagenet
+  path: /datasets/imagenet
+```
+</div>
+</div>
 
 
 
 
 
-In addition to override `hydra.searchpath` in primary config, one can also override from the command line.
+`hydra.searchpath` can be defined or overridden via the command line as well:
 
 ```bash title="command line override"
 python my_app.py 'hydra.searchpath=[pkg://additonal_conf]'
 ```
 
-#### Overriding `--config-dir` from the command line
-This is a less flexible alternative to `hydra.searchpath`. 
-See this [page](/docs/advanced/hydra-command-line-flags) for more info.
```

Comment:
Don't remove it yet.
As long as we didn't deprecate it it should be documented here.